src/kubernetes.cc

```diff
@@ -796,6 +796,15 @@ void KubernetesReader::WatchMaster(
   if (verbose) {
     LOG(INFO) << "WatchMaster(" << name << "): Contacting " << endpoint;
   }
+  std::mutex last_received_mutex;
+  auto last_received = std::chrono::high_resolution_clock::now();
+  if (health_checker_) {
+    health_checker_->RegisterCallback(name, [&last_received_mutex, &last_received]{
```

Comment:
Why not capture `last_received` by const reference?

Reply:
Looks like capturing by const reference is not available in C++11.